_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -492,9 +495,7 @@ public static function get_plans( $request ) {
 	 * @return array An array of jitms
 	 */
 	public static function get_jitm_message( $request ) {
-		require_once( JETPACK__PLUGIN_DIR . 'class.jetpack-jitm.php' );
-
-		$jitm = Jetpack_JITM::init();
+		$jitm = new JITM_Manager();
```

Comment:
I think packages should be responsible of registering their API endpoints. 
I would move all this code to the package, what do you all think?

Reply:
Sounds good, I will do it in the next iteration.